### ISSUE TITLE
WinAppDriver record video

### ISSUE THREAD

**Issue by RahulDixit2021** [microsoft/WinAppDriver#1482]
Hi,

How do I record a video of my winappdriver tests? None of the appium screen recording apis are supported.

Thanks,
Rahul

**Comment by anunay1** [comment]
No, its not supported, I created my custom recorder and recording the tests? You can implement desktop recorder as part of your framework.

**Comment by liljohnak** [comment]
We did this using MSTestv2, by extending the TestClass.

**Comment by shoaibmansoor** [comment]
Hi @RahulDixit2021 I haven't tried this feature myself but Appium indeed support video recording. Please refer: https://github.com/appium/appium-windows-driver

**Comment by Shakevg** [comment]
@RahulDixit2021 
What is language binding are you using?
E.g. You can use one of the tools:
- Use *.runsettings file for running tests with enabled video recording https://docs.microsoft.com/en-us/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2019#videorecorder-data-collector
- Custom recording 
  - https://stackoverflow.com/questions/57424616/how-to-record-test-execution-using-selenium-c-sharp-in-netcore
  - https://medium.com/maestral-solutions/setup-automation-test-screen-recorder-with-selenium-webdriver-and-c-aee8b6d1ba9b
